umm-spec-lib/resources/xml-schemas/echo10/Collection.xsd

```diff
@@ -260,7 +260,27 @@ xmlns:xs="http://www.w3.org/2001/XMLSchema">
             <xs:enumeration value="NEAR_REAL_TIME">
               <xs:annotation>
                 <xs:documentation> Data from the source that are
-                available for use within a time that is short in
+                available for use within a time that is short(1-3 hours) in
+                comparison to important time scales in the phenomena
+                being studied. This data is not science quality and
+                is not retained by EOSDIS once the SCIENCE_QUALITY
+                product is archived </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="LOW_LATENCY">
+              <xs:annotation>
+                <xs:documentation> Data from the source that are
+                available for use within a time that is short(4-24 hours) in
+                comparison to important time scales in the phenomena
+                being studied. This data is not science quality and
+                is not retained by EOSDIS once the SCIENCE_QUALITY
+                product is archived </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="EXPEDITED">
+              <xs:annotation>
```

Comment:
On line 246 -the documentation string for CollectionDataType, can you please change the description to the following: 
This element is used to identify the collection's ready for end user consumption latency from when the data was acquired by an instrument. NEAR_REAL_TIME is defined to be ready for end user consumption 1 to 3 hours after data acquisition. LOW_LATENCY is defined to be ready for consumption 3 to 24 hours after data acquisition. EXPEDITED is defined to be 1 to 4 days after data acquisition. SCIENCE_QUALITY is defined to mean that a collection has been fully and completely processed which usually takes between 2 to 3 weeks after data acquisition. OTHER is defined for collection where the latency is between EXPEDITED and SCIENCE_QUALITY.

On line 255 the documentation string for SCIENCE_QUALITY can you replace the text with the following:
All data products that have been fully and completely processed which usually takes between 2 to 3 weeks after data acquisition.

On line 262 the documentation string for NEAR_REAL_TIME can you replace the text with the following: 
The data is ready for end user consumption 1 to 3 hours after data acquisition. This data is not fully processed and is not retained by EOSDIS once the SCIENCE_QUALITY product is archived.

On the new line 272 the documentation for LOW_LATENCY can you replace the text with the following:
The data is ready for end user consumption 3 to 24 hours after data acquisition. This data is not fully processed and is not retained by EOSDIS once the SCIENCE_QUALITY product is archived.